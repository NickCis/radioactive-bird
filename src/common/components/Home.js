import React from 'react';
import NotIdealState from './NotIdealState';
import Search from 'material-ui-icons/Search';

export const Home = () => (
  <NotIdealState
    headline="Search tweets!"
    subheading="Use the upper Search bar"
    Icon={Search}
  />
);

export default Home;
