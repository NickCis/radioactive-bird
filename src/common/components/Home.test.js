import React from 'react';
import { createRender } from 'material-ui/test-utils';
import Home from './Home';

describe('<Home />', () => {
  const render = createRender();

  it('renders correctly', () => {
    const wrapper = render(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
