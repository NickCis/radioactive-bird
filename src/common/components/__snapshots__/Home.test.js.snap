// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders correctly 1`] = `
<div
  class="NotIdealState-wrapper-1"
>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root-5 NotIdealState-icon-2"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <path
      d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
    >
      
    </path>
  </svg>
  <h1
    class="MuiTypography-root-6 MuiTypography-display1-10 NotIdealState-headline-3"
  >
    Search tweets!
  </h1>
  <h3
    class="MuiTypography-root-6 MuiTypography-subheading-13 NotIdealState-subheading-4"
  >
    Use the upper Search bar
  </h3>
</div>
`;
